delete.php

```diff
@@ -1,21 +1,16 @@
 <?php
 
-	include('config/db_connection.php');
-	include('config/photo_path.php');
+session_start();
+require_once('config/photo_path.php');
```

Comment:
Please remove photo_path.php as you have image path constant in constants.php file